### PR TITLE
[Bugfix/LIVE-8914]: Info message not triggered when user accesses a coin from the Market tab

### PR DESCRIPTION
**Pull request by kallen-ledger**
### 📝 Description

* Add `onPressWhenDisabled` prop to native ui component.
* Market detail screen pop will now occur when user clicks on "disabled" looking button.

### ❓ Context

- **Impacted projects**: `LLM, UI` 
- **Linked resource(s)**: [LIVE-8914](https://ledgerhq.atlassian.net/browse/LIVE-8914) 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo
https://github.com/LedgerHQ/ledger-live/assets/132384348/40004e82-80d2-4d49-91c4-9d5007a73dfb

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-8914]: https://ledgerhq.atlassian.net/browse/LIVE-8914?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/ui/packages/native/src/components/cta/Button/index.tsx [hunk 1]
```diff
@@ -33,12 +34,21 @@ const IconContainer = styled.View<{
     p.iconButton ? "" : p.iconPosition === "left" ? `margin-right: 10px;` : `margin-left: 10px;`}
 `;
 
-export const Base = baseStyled(TouchableOpacity).attrs<ButtonProps>((p) => ({
-  ...getButtonColorStyle(p.theme.colors, p).button,
-  // Avoid conflict with styled-system's size property by nulling size and renaming it
-  size: undefined,
-  sizeVariant: p.size,
-}))<
+export const Base = baseStyled(TouchableOpacity).attrs<ButtonProps>((p) => {
```
Comment (sarneijim) [inline libs/ui/packages/native/src/components/cta/Button/index.tsx:37]: If design system is happy with this change, I think we need to apply the same change to libs/ui/package/react.

Comment (kallen-ledger) [reply libs/ui/packages/native/src/components/cta/Button/index.tsx:37]: Sent them a message on slack, so just waiting to see what they say 👍

**Review by ofreyssinet-ledger — APPROVED** [review]
lgtm 👍 nice addition

**Comment by KVNLS** [comment]
Hi @kallen-ledger, thank you for the improvement!
Do you have some Figma about it, please?
We have a similar use case for the NFT send action on iOS that is solved with a drawer, if not already done could you confirm with the design if we want to do it this way? Thanks
image

**Comment by kallen-ledger** [comment]
> We have a similar use case for the NFT send action on iOS that is solved with a drawer, if not already done could you confirm with the design if we want to do it this way? Thanks

Hey @KVNLS. I don't have a figma file for this as it was only done as part of a bugfix ticket. If you'd like I could add this to the storybook?

File: libs/ui/packages/native/src/components/cta/Button/index.tsx (hunk 1 above)
Comment (sarneijim) [reply libs/ui/packages/native/src/components/cta/Button/index.tsx:37]: Super. I think as Olivier has approve it they are mostly happy, please do not forget align LLD component any basic design will be good on my side but you can ask Diego.

**Comment by KVNLS** [comment]
> > We have a similar use case for the NFT send action on iOS that is solved with a drawer, if not already done could you confirm with the design if we want to do it this way? Thanks
> 
> Hey @KVNLS. I don't have a figma file for this as it was only done as part of a bugfix ticket. If you'd like I could add this to the storybook?

No, I'm fine with it, I think that you can merge this PR, congrats! If the design wants to change the behavior they will create a dedicated task

**Pull request merged**